detectors/node/resource-detector-git/src/fecth-git-data.ts

```diff
@@ -0,0 +1,15 @@
+import child_process from 'child_process';
+import path from 'path';
+import fs from 'fs';
+
+export const executeGitCommand = (gitCommand: string): string => {
+    return child_process
+        .execSync(gitCommand, { stdio: ['ignore', 'pipe', 'pipe'] })
```

Comment:
Maybe add `timeout` option for execSync? 
@blumamir

Reply:
good idea 👍